packages/contracts/test/contracts/unichain/DataUnionTemplate.test.ts

```diff
@@ -120,7 +121,7 @@ describe("DataUnionTemplate", () => {
 
         expect(await dataUnionSidechain.totalEarnings()).to.equal(2400)
         expect(await dataUnionSidechain.totalAdminFees()).to.equal(300)
-        expect(await dataUnionSidechain.totalDataUnionFees()).to.equal(300)
+        expect(await dataUnionSidechain.totalProtocolFees()).to.equal(300)
```

Comment:
We should probably also check the earnings of the owner and fee beneficiary. Currently we're just checking that the cumulative counters have been updated.
```
        expect(await dataUnionSidechain.getEarnings(duBeneficiary.address).to.equal(300)
        expect(await dataUnionSidechain.getEarnings(creator.address).to.equal(300)
```